src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -789,6 +789,9 @@
   - name: Trigger a task run by overriding the context and file passed during Task create.
```

Comment:
Should this add something like the following?
```suggestion
  - name: Trigger a task run by overriding the context and file passed during Task create with a remote repository
```

Not sure the correct wording / task context lingo. But suggesting this change, given the example you are adding is showing how to override the context with a local path and this existing is overriding with a remote repo / path...

Please choose a better wording, than the suggestion I made, the point is we should explain what kind of context we are overrideing the original one with.